lua/nvim-web-devicons/icons-default.lua

```diff
@@ -446,6 +674,18 @@ local icons_by_filename = {
 }
 
 local icons_by_file_extension = {
+  ["3gp"] = {
+    icon = "",
```

Comment:
I know this hasn't been communicated however we are moving towards seti icons as defaults, with the intent of adding a material design alternative.

Let's change these videos to nf-seti-video please.

Reply:
ok, 7  substitutions